website/docs/r/policy_transport_zone.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+subcategory: "Fabric"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_transport_zone"
+description: A resource to configure Policy Transport Zone.
+---
+
+# nsxt_policy_transport_zone
+
+This resource provides a method for the management of Policy based Transport Zones (TZ). A Transport Zone defines the scope to which a network can extend in NSX. For example an overlay based Transport Zone is associated with both hypervisors and logical switches and defines which hypervisors will be able to serve the defined logical switch. Virtual machines on the hypervisor associated with a Transport Zone can be attached to logical switches in that same Transport Zone.
```

Comment:
I think we should use policy terminology here (segment rather than logical switch)

Reply:
done